### PR TITLE
[REFACTOR] CSS for PostCard.tsx

### LINKED ISSUE #2895: Refactor CSS in `src/components/UserPortal/PostCard/PostCard.tsx`
# Refactor CSS in this file

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

This file contains embedded CSS references that should be completely moved to:
- `src/style/app.module.css`

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is for this file to use this new design pattern. This means that:
1. All embedded CSS in this file must be merged into the global CSS file using pre-existing CSS.
1. All embedded CSS in this file must be removed

### PR DESCRIPTION
**Pull request by hustlernik**
**What kind of change does this PR introduce?**

Streamlined all CSS for PostCard.tsx into a single global file, fixed UI bugs on the PostCard.tsx , and ensured no conflicts with other pages.

**Issue Number:**

Fixes #2895 

**Did you add tests for your changes?**

No

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Introduced new styling classes for the post card interface, enhancing layout and visual hierarchy.
	- Added a new CSS variable for input area background color.
- **Bug Fixes**
	- Removed outdated CSS file, streamlining the styling process for the post card component.
- **Refactor**
	- Updated class names in the PostCard component for improved clarity and specificity of styles.

### CODE REVIEW

**Review by palisadoes — DISMISSED** [review]

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by hustlernik** [comment]
@palisadoes okay, doing.

File: src/style/app.module.css [hunk 1]
```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
...
+  gap: 1rem;
+  margin: 5px 0px;
+}
+
+.textModal {
+  margin-top: 10px;
+}
+
+.colorPrimary {
+  background: #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:2483]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: src/style/app.module.css (hunk 1 above)
Comment (hustlernik) [reply src/style/app.module.css:2483]: done.

File: src/style/app.module.css [hunk 2]
```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
...
+.cardHeaderPostCard {
+  display: flex;
+  width: 100%;
+  padding-inline: 0;
+  padding-block: 0;
+  flex-direction: row;
+  gap: 0.5rem;
+  align-items: center;
+  background-color: white;
+  border-bottom: 1px solid #dddddd;
```
Comment (palisadoes) [inline src/style/app.module.css:2352]: Convert to a variable. There should be some preexisting ones, if not please create it.

File: src/style/app.module.css [hunk 3]
```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
...
+  gap: 1rem;
+  margin: 5px 0px;
+}
+
+.textModal {
+  margin-top: 10px;
+}
+
+.colorPrimary {
+  background: #7c9beb;
```
Comment (palisadoes) [inline src/style/app.module.css:2483]: Convert to a variable. There should be some preexisting ones, if not please create it.

File: src/style/app.module.css [hunk 4]
```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
...
+  padding-bottom: 1rem;
+}
+
+.modalFooter {
+  background-color: white;
+  width: 100%;
+  padding-block: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  border-top: 1px solid #dddddd;
```
Comment (palisadoes) [inline src/style/app.module.css:2500]: Convert to a variable. There should be some preexisting ones, if not please create it.

File: src/style/app.module.css [hunk 5]
```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
...
+}
+
+.inputArea {
+  border: none;
+  outline: none;
+  background-color: #f1f3f6;
```
Comment (palisadoes) [inline src/style/app.module.css:2506]: Convert to a variable. There should be some preexisting ones, if not please create it.

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by palisadoes** [comment]
Please make code rabbit approve your changes

**Comment by palisadoes** [comment]
Please fix the failing test

File: src/style/app.module.css (hunk 2 above)
Comment (palisadoes) [reply src/style/app.module.css:3545]: This isn't fixed

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Review by palisadoes — CHANGES_REQUESTED** [review]
This file should not be included.

**Comment by hustlernik** [comment]
@palisadoes I have messed stuffs a bit, so will close the PR if needed and raise another in sometime.

**Pull request closed without merging**